### PR TITLE
Additional style changes

### PR DESCRIPTION
**Pull request by nathan-weinberg**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -128,8 +96,45 @@ def get_jenkins_jobs(server, job_search_fields):
...
+		if bug_id == 0:
+			bugs[0] = {'bug_name': 'No bug on file', 'bug_url': None}
+			continue
+
+		# get bug info from bugzilla API
+		try:
+
+			# hotfix: API call does not work if '/' present at end of URL string
+			parsed_bz_url = config['bz_url'].rstrip('/')
+
```
Comment (EliadCohenRH) [inline main.py]: Line break intentional?

File: main.py [hunk 2]
```diff
@@ -128,8 +96,45 @@ def get_jenkins_jobs(server, job_search_fields):
...
+		# 0 should be default in YAML file (i.e. no bugs recorded)
+		# if present reference should be made in bug dict
+		if bug_id == 0:
+			bugs[0] = {'bug_name': 'No bug on file', 'bug_url': None}
+			continue
+
+		# get bug info from bugzilla API
+		try:
+
+			# hotfix: API call does not work if '/' present at end of URL string
```
Comment (EliadCohenRH) [inline main.py]: Is this comment still needed?

File: main.py [hunk 3]
```diff
@@ -128,8 +96,45 @@ def get_jenkins_jobs(server, job_search_fields):
...
+
+		# 0 should be default in YAML file (i.e. no bugs recorded)
+		# if present reference should be made in bug dict
+		if bug_id == 0:
+			bugs[0] = {'bug_name': 'No bug on file', 'bug_url': None}
+			continue
+
+		# get bug info from bugzilla API
+		try:
+
```
Comment (EliadCohenRH) [inline main.py]: Remove this line break?

File: main.py (hunk 2 above)
Comment (nathan-weinberg) [reply main.py]: 'get bug info' or hotfix comment?

File: main.py (hunk 1 above)
Comment (nathan-weinberg) [reply main.py]: Yeah I have this line isolated since it is a hotfix and may need to be changed at a later date

File: main.py (hunk 3 above)
Comment (nathan-weinberg) [reply main.py]: See above comment

File: main.py (hunk 2 above)
Comment (EliadCohenRH) [reply main.py]: hotfix comment

File: main.py (hunk 3 above)
Comment (EliadCohenRH) [reply main.py]: The line break right after "try:"

File: main.py (hunk 1 above)
Comment (EliadCohenRH) [reply main.py]: Got it

**Pull request merged**